puppet/modules/jenkins_job_builder/manifests/config.pp

```diff
@@ -39,18 +37,15 @@
     timeout     => $jenkins_jobs_update_timeout,
     path        => '/bin:/usr/bin:/usr/local/bin',
     refreshonly => true,
```

Comment:
Should this be removed so it's triggered on every run? Or what would trigger it?